src/main/java/gwt/material/design/addins/client/fileuploader/MaterialFileUploader.java

```diff
@@ -20,649 +20,727 @@
+			setInitialize(true);
+		}
+	}
+
+	@Override
+	public void add(Widget child) {
+		super.add(child);
+	}
+
+	public void initDropzone() {
+		String previews = DOM.createUniqueId();
+		uploadPreview.getUploadCollection().setId(previews);
+		if (clickable.isEmpty()) {
+			String clickable = DOM.createUniqueId();
+			if (getWidget(1) instanceof MaterialUploadLabel) {
+				MaterialUploadLabel label = (MaterialUploadLabel) getWidget(1);
+				label.getIcon().setId(clickable);
+			} else {
+				getWidget(1).getElement().setId(clickable);
+			}
+			setClickable(clickable);
+		}
+		if (!isPreview()) {
+			uploadPreview.setDisplay(Display.NONE);
+		}
+		initDropzone(getElement(), uploadPreview.getUploadCollection().getItem().getElement(), previews, uploadPreview.getElement(), uploadPreview.getUploadHeader().getUploadedFiles().getElement(),
+				getUrl(), getMaxFileSize(), getMaxFiles(), getMethod().getCssName(), isAutoQueue(), getAcceptedFiles(), getClickable(), preview, isWithCredentials());
+	}
+
+	/**
+	 * Intialize the dropzone component with element and form url to provide a
+	 * dnd feature for the file upload
+	 * 
+	 * @param e
+	 * @param url
+	 */
+	private native void initDropzone(Element e, Element template, String previews, Element uploadPreview, Element uploadedFiles, String url, int maxFileSize, int maxFiles, String method,
+			boolean autoQueue, String acceptedFiles, String clickable, boolean preview, boolean withCredentials) /*-{
+																													var that = this;
+																													$wnd.jQuery(document).ready(function() {
+																													var previewNode = $wnd.jQuery(template);
+																													var previewContainer = $wnd.jQuery("#" + previews).html();
+																													var globalResponse = "";
+																													previewNode.id = "";
+																													var previewTemplate = previewNode.parent().html();
+
+																													var totalFiles = 0;
+																													var zdrop = new $wnd.Dropzone(e, {
+																													url: url,
+																													maxFilesize: maxFileSize,
+																													method: method,
+																													maxFiles: maxFiles,
+																													previewTemplate: previewTemplate,
+																													acceptedFiles: acceptedFiles,
+																													autoQueue: autoQueue,
+																													previewsContainer: "#" + previews,
+																													clickable: "#" + clickable,
+																													withCredentials: withCredentials
+																													});
+
+																													zdrop.on('drop', function () {
+																													that.@gwt.material.design.addins.client.fileuploader.MaterialFileUploader::fireDropEvent()();
+																													if(preview) {
+																													$wnd.jQuery(e).removeClass("active");
+																													}
+																													});
```

Comment:
The problem is that my eclipse is doing this spaces here. I do not know how to change it. I would like to use same ide and same configuration as You use. Can You share spec of Your IDE?